app/controllers/LockableS3ObjectController.scala

```diff
@@ -40,7 +39,7 @@ abstract class LockableS3ObjectController[T : Decoder : Encoder](
   dataObjectSettings: S3ObjectSettings,
   fastlyPurger: Option[FastlyPurger],
   runtime: DefaultRuntime
-)(implicit ec: ExecutionContext) extends AbstractController(components) with Circe with StrictLogging {
+)(implicit ec: ExecutionContext) extends AbstractController(components) with Circe {
```

Comment:
I assume `StrictLogging` isn't being used?

Reply:
`logger` is now available via `AbstractController`

Reply:
Got it